docs/reference/cli.md

```diff
@@ -8,28 +8,50 @@ Aerleon contains three command line programs:
 
 [`pol2yaml`](#pol2yaml) converts .pol, .inc, .svc, and .net files to equivalent YAML files. It can be found in [its own repository](https://github.com/aerleon/pol2yaml) but is documented here.
 
-### Config Files
+
+
+## Common Options
+
+Options used by more than one command line program are listed here. Setting these options in your config file is a good practice.
```

Comment:
Possibly link to where we have instructions on setting the config file.

Reply:
Good idea - done